architecture/understanding-development.adoc

```diff
@@ -1,45 +1,396 @@
+These images are based on Red Hat Enterprise Linux and are similar to base
+images that Red Hat has offered in the past, with one major difference: they
+are freely redistributable without a Red Hat subscription. As a result, you can
+build your application on UBI images without having to worry about how they
+are shared or the need to create different images for different environments.
+
+These UBI images have standard, init, and minimal versions. You can also use the
+link:https://access.redhat.com/documentation/en-us/red_hat_software_collections/3/html-single/using_red_hat_software_collections_container_images/index[Red Hat Software Collections]
+images as a foundation for applications that rely on specific runtime
+environments such as Node.js, perl, or python. Special versions of some of
+these runtime base images referred to as Source-to-image (S2I) images. With
+S2I images, you can insert your code into a base image environment that is ready
+to run that code.
+
+S2I images are available for you to use directly from the {product-title} web UI
+by selecting *Catalog* -> *Developer Catalog*, as shown in the following figure:
+
+.Choose S2I base images for apps that need specific runtimes
+image::developer-catalog.png[{product-title} Developer Catalog]
+
+[id="choosing-registry"]
+=== Choosing a registry
+
+Container Registries are where you store container images so you can share them
+with others and make them available to the platform where they ultimately run.
+You can select large, public container registries that offer free accounts or a
+premium version that offer more storage and special features. You can also
+install your own registry that can be exclusive to your organization or
+selectively shared with others.
+
+To get Red Hat images and certified partner images, you can draw from the
+Red Hat Registry. The Red Hat Registry is represented by two locations:
+`registry.access.redhat.com`, which is unauthenticated and deprecated, and
+`registry.redhat.io`, which requires authentication. You can learn about the Red
+Hat and partner images in the Red Hat Registry from the
+link:https://access.redhat.com/containers/[Red Hat Container Catalog].
+Besides listing Red Hat container images, it also shows extensive information
+about the contents and quality of those images, including health scores that are
+based on applied security updates.
+
+Large, public registries include link:https://hub.docker.com/[Docker Hub] and
+link:https://quay.io/[Quay.io]. The Quay.io registry is owned and managed by Red
+Hat. Many of the components used in {product-title} are stored in Quay.io,
+including container images and the Operators that are used to deploy
+{product-title} itself. Quay.io also offers the means of storing other types of
+content, including Helm Charts.
+
+If you want your own, private container registry, {product-title} itself
```

Comment:
This feels more focused on Quay than the OpenShift Image Registry, we may want to remove or move some of the Quay content. Though, this is not at all urgent and something that can be addressed later.

Reply:
@chrisnegus, thoughts?